prijateli_tree/app/routers/games.py

```diff
@@ -201,6 +216,19 @@ def view_round(game_id: int, player_id: int, db: Session = Depends(get_db)):
             status_code=HTTPStatus.NO_CONTENT, detail="game not found"
         )
 
+    player = None
+    for p in game.players:
+        if p.id == player_id:
+            player = db.query(User).filter_by(id=p.user_id).one_or_none()
+
+    if player is None:
+        raise HTTPException(
+            status_code=HTTPStatus.NOT_FOUND, detail="player not in game"
+        )
+
+    # Here's where you can get the correct text for your templating.
+    # template_text = languages[player.language.abbr]
```

Comment:
Here's how you get the correct language text for the templates, @FedericoDM. I inserted the code into two endpoints.